AUTHORS

```diff
@@ -1,3 +1,215 @@
+Ibrahem Zidan <brahem@aotsw.com>
+Ibrahim Hossam <ibrahem.h.basyone@gmail.com>
+Ibrahim Zidan <brahem@aotsw.com>
+inhyositsu <inhyositsu@gmail.com>
+Inzianity <inzianity@gmail.com>
+jaBote <jaBote@fake-emails-rock.com>
+jaBote <jaBote@users.noreply.github.com>
+jaBote <javote91@hotmail.com>
+jaBote <j@bot.e>
+Jackson <chadfield.jackson@gmail.com>
+Jedzkie <jedzkie13@rocketmail.com>
+jmanfffreak <jmish1987@gmail.com>
+Jônatas Andreta <jonataandretta@hotmail.com>
+Jorge C <ctt@csnv.es>
+Jose Luis Rivera Flores <jose-pro_h4kr@hotmail.com>
+j-tkay <joseph.tk.ea@gmail.com>
+kenpachi2k11 <Kenpachi.Developer@gmx.de>
+Kenpachi2k13 <3476227+Kenpachi2k13@users.noreply.github.com>
+Kenpachi2k13 <Kenpachi.Developer@gmx.de>
+Kenpachi Developer <Kenpachi.Developer@gmx.de>
+Kenpachi <Kenpachi.Developer@gmx.de>
+Kerubael <keru@ro-mina.net>
+KirieZ <guilherme_kiriez@hotmail.com>
+KirieZ <guilherme.menaldo@outlook.com>
+kisuka <kisuka@kisuka.com>
+Kolkka <lmiranda@lumigo.net>
+Kpy! <ouroboros.ai@gmail.com>
+landergate <landergate@landergate.com>
+lemongrass3110 <lemongrass@kstp.at>
+Leo Pflug <skyleo@skyleo.de>
+linton-dawson <pushkarkukde@gmail.com>
+LiYu <cairoliyu@gmail.com>
+louisplag <42522191+louisplag@users.noreply.github.com>
+Lucas Brito <lucasljsb@gmail.com>
+M45T3Ryu <jose-pro_h4kr@hotmail.com>
+malufett <malufett.eat.my.binaries@gmail.com>
+maqc1 <alex--223@hotmail.com>
+Marcelo Liberato <marceloll@outlook.com>
+Mark Hester <Marky360@live.ie>
+marky291 <Marky360@live.ie>
+Marvin Lamart <m.lamart@agence-passerelle.com>
+masao87 <masao@gmx.ch>
+Masao <masao@gmx.ch>
+MasterOfMuppets <masterofmuppets89@gmail.com>
+Matej Pristak <matej.pristak@gmail.com>
+Matheus Macabu <mkbu95@gmail.com>
+Mathy <MathyM@users.noreply.github.com>
+Matias <matiassw@gmail.com>
+Maytichai Saowa <nightcoremax@gmail.com>
+mekolat <git@gumi.ca>
+Mhalicot <sevenzz23@yahoo.com>
+michaelforge <landergate@landergate.com>
+michieru <michieru-kun@hotmail.com>
+Milchdealer <jocker300@hotmail.de>
+Milk <welcomerooot@gmail.com>
+milky-milk <welcomerooot@gmail.com>
+Miniack <Redemer_Braulio@hotmail.com>
+moguri85 <ikke85@gmail.com>
+momacabu <macabu.matheus@gmail.com>
+MordekaiserGod <mordekaiser@iwillgetaproperemail.com>
+MrKeiKun <the.keikun@gmail.com>
+Mr.Postman <MrPostman@windowslive.com>
+Mumbles <datmumbles@icloud.com>
+Mumbles <icelestial@outlook.com>
+Mumbles <mumbles@qt-ro.com>
+Mumbles <mumbles@rhythm.ro>
+Murilo_Bio <murilopereti@gmail.com>
+Murilo Pereti Tavares <murilopereti@gmail.com>
+Mysteries <mysteriousragnarok@hotmail.com>
+Mysterious <mysteriousragnarok@hotmail.com>
+nadiyama <ctt@csnv.es>
+nadiyama <jorgecsnova@gmail.com>
+Napster <arokaice@live.com>
+Nihadm89 <nihadm89@hotmail.com>
+Noil <senpainoil@gmail.com>
+Normynator <Norman.Ziebal@web.de>
+ossi0110 <asmodus1@web.de>
+ossi0110 <maikkempkes@alice.de>
+panikon <panikon@zoho.com>
+Paradox924x <paradox924x@gmail.com>
+Patskie <escalona3413@gmail.com>
+Phong Truong <phongtruongit@gmail.com>
```

Comment:
this is my personal email, can you change it to Jenkijo <jenkijo@hrzn.community>?

Reply:
This list is auto generated authors from git commits.

if you want email mapping, add them to ``.mailmap``
git will automatically use only given email if see other your emails.